tests/data/validate_mbz/questions.xml

```diff
@@ -0,0 +1,122 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Minor: For consistency in the directory naming here, we should have this be `tests/data/validate_mbz_html/questions.xml` (e.g. where the directory under `data/` matches the corresponding script name).